generator/uscore/uscore_generator.rb

```diff
@@ -740,15 +786,26 @@ def get_search_params(search_parameters, sequence, grab_first_value = false)
 
         if search_param_string.include? 'get_value_for_search_param'
           search_param_value_check =
-            %(search_params.each { |param, value| skip "Could not resolve \#{param} in given resource" if value.nil? }\n)
-          return [search_param_string, search_param_value_check].join('')
+            %(
+              if search_params.any? { |param, value| value.nil? }
```

Comment:
```suggestion
              if search_params.any? { |_param, value| value.nil? }
```
This could also be `if search_params.value? nil`, but I think that is less clear.